chapter12/mdw/Ex-02.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+my $oldest_file;
+my $oldest_age;
+
+foreach my $filename (@ARGV){
+	if ( -e $filename ){
+		if ( -A $filename > $oldest_age ){
```

Comment:
Yeah, I think you could get away with wither creation or modification date here... but not the last access date.